apps/govquests-api/govquests/action_tracking/lib/action_tracking/services.rb

```diff
@@ -0,0 +1,31 @@
+module ActionTracking
+  module Services
+    module AgoraService
+      def get_delegate(address)
```

Comment:
👀

Reply:
this service should not be here, but the idea before was to just define the interface that those services would need to have